include/partition/multi_level_partition.hpp

```diff
@@ -314,11 +323,18 @@ template <bool UseShareMemory> class MultiLevelPartitionImpl final
         }
     }
 
-    //! this is always owned by this class because it is so small
-    LevelData level_data;
+    LevelDataPtr level_data;
```

Comment:
Hrm this doesn't work. `= {}` seems to create an empty unique ptr. I need a unique ptr with a real value in it.